docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>This is definitely going to be better with tqdm as a progress bar. I'll go back to that during code review.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I agree that would be nicer - the problem is that the simulation loop is actually a while loop, so I don't know ahead of time how many iterations to expect. Is there a way to modify tqdm to print the current number of particles on the grid, like I'm doing now, as a measure of progress?

Reply:
Yeah! I'll add that when I'm back to code review for this, okay? :)

Reply:
Sure, that'd be great :)